pyrenew/convolve.py

```diff
@@ -168,9 +168,9 @@ def _new_scanner(
 
 
 def compute_delay_ascertained_incidence(
-    incidence_to_observation_rate: ArrayLike,
+    p_observed_given_incident: ArrayLike,
     latent_incidence: ArrayLike,
-    incidence_to_observation_delay_interval: ArrayLike,
+    delay_incidence_to_observation_delay_pmf: ArrayLike,
```

Comment:
Can strike the second `delay`
```suggestion
    delay_incidence_to_observation_pmf: ArrayLike,
```

Reply:
fixed in https://github.com/CDCgov/PyRenew/pull/398/commits/4bc1736d0e1fd2485975cb9d18a44e021896fa8d